_i18n/en/general/events-add-calendar-modal.md

```diff
@@ -0,0 +1,5 @@
+Always want to have the latest communinity call dates and times in your calendar? Then copy the URL and add it as a remote calendar in your calendar app.
+
+(For the moment we don't have the option add individual events to your calendar.)
```

Comment:
I would remove that. Explicitly specifying what we don't support is just overhead for readers.